pyproject.toml

```diff
@@ -45,7 +45,8 @@ covalent = ["covalent>=0.234.1-rc.0; platform_system!='Windows'", "covalent-clou
 dask = ["dask[distributed]>=2023.12.1", "dask-jobqueue>=0.8.2"]
 defects = ["pymatgen-analysis-defects>=2024.10.22", "shakenbreak>=3.2.0"]
 jobflow = ["jobflow>=0.1.14", "jobflow-remote[gui]>=0.1.0"]
-mlp = ["matgl>=1.1.2", "chgnet>=0.3.3", "mace-torch>=0.3.3", "torch-dftd>=0.4.0", "sevenn>=0.10.1", "orb-models>=0.4.1"]
+mlp1 = ["chgnet>=0.3.3", "torch-dftd>=0.4.0", "sevenn>=0.10.1", "orb-models>=0.4.1", "fairchem-core>=1.4.0"]
+mlp2 = ["mace-torch>=0.3.3", "matgl>=1.1.2"]
```

Comment:
are these dep sets split based on required `e3nn` version? maybe worth renaming to something more descriptive?

Reply:
It's a mess 😅 matgl and chgnet have torch compatibility issues, and then there are also e3nn version issues too. I definitely should rename them though!